dspace-api/src/main/java/org/dspace/authenticate/ShibAuthentication.java

```diff
@@ -332,6 +328,8 @@ public int authenticate(Context context, String username, String password,
 							affiliation = affiliation.substring(index+1, affiliation.length());
 						}
 					} 
+                                        // Replace spaces with underscores
+					affiliation = affiliation.replace(" ", "_");
```

Comment:
This is indeed a breaking change.

You can also escape spaces in a property name using Java Unicode. So to match with the "university member" role, you should define a property: `role.university\u0020member = Member`